src/main/functions/DidJWT.ts

```diff
@@ -150,7 +136,7 @@ async function signDidJwtExternal(
 }
 
 async function signDidJwtSupplied(
-  payload: AuthenticationRequestPayload | AuthenticationResponsePayload,
+  payload: IdTokenPayload,
```

Comment:
Why is what is a generic signing method for a request/response payload all of a sudden removing support for one of these?

Reply:
Fixed